Pipfile

```diff
@@ -33,6 +33,11 @@ click = ">=7.0"
 colorama = "*"
 boto3 = "*"
 flask-limiter = "*"
+jinja2 = ">=2.10.1"
+dash = "==0.34.0"
+dash-core-components = "==0.41.0"
+dash-html-components = "==0.13.4"
+dash-dangerously-set-inner-html = "==0.0.2"
```

Comment:
Hmm, I'm a little concerned with some of these deps, like this one.  Whats the story here?

Reply:
This was used in `status_page.py` for the fleet state icon. 
It's basically a way to specify html within dash.

The `fleet_state_icon` property of the node_dict returns html - and the individual values cannot be obtained through object calls.